src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write code here
+const listElement = document.querySelector('ul');
+
+const parseSalary = (salary) => {
+  const parsedSalary = salary.replace('$', '').replace(',', '');
+
+  try {
+    return Number(parsedSalary);
+  } catch (error) {
+    return 0;
+  }
+};
+
+const sortList = (list) => {
+  const listElements = Array.from(list.children);
+
+  listElements.sort((a, b) =>
+    parseSalary(b.dataset.salary) - parseSalary(a.dataset.salary));
+
+  list.innerHTML = '';
+
+  listElements.forEach((element) => list.appendChild(element));
```

Comment:
```suggestion
  listElements.forEach(list.appendChild);
```

Reply:
Unfortunately, it does not work as expected, it is possible to `.bind(list)` so your solution will work, but, for me, It makes solution harder to understand.